src/record/export/usecases/get.ts

```diff
@@ -13,6 +13,10 @@ import { existsSync, mkdirSync, writeFileSync } from "fs";
 import { getAllRecords } from "./get/getAllRecords";
 import type { LocalRecordRepository } from "./interface";
 import { replaceSpecialCharacters } from "../utils/file";
+import { logger } from "../../../utils/log";
+
+export const NO_ANY_RECORDS_WARNING =
+  "The specified app does not have any records.";
```

Comment:
MUST:
This warning message is shown not only in the situation the app has no record but also no record matched with the specified `condition` option.
So please update the message like "No records exist in the app or match the condition."